streamer/src/nonblocking/quic.rs

```diff
@@ -702,17 +718,41 @@ async fn handle_connection(
     );
     let stable_id = connection.stable_id();
     stats.total_connections.fetch_add(1, Ordering::Relaxed);
+    let mut next_interval = tokio::time::Instant::now()
```

Comment:
I think the code can made a little clearer. Have a constant of the interval duration (100ms), record the last reset time (last_reset), if duration(now-last_reset), do reset.

Reply:
I think we can delay doing this math until after we get a stream: 743, if we timeout waiting for a stream, we do not need to do this. Also we can wrap this in a function